src/domain/accounts/services/src/authentication_service_impl.rs

```diff
@@ -136,16 +129,50 @@ impl AuthenticationServiceImpl {
     pub fn decode_access_token(
         &self,
         access_token: &str,
-    ) -> Result<TokenData<KamuAccessTokenClaims>, AccessTokenError> {
+    ) -> Result<AccessTokenArg, AccessTokenError> {
+        if &access_token[..2] == ACCESS_TOKEN_PREFIX {
+            return self
+                .access_token_svc
+                .decode_access_token(access_token)
+                .map_err(|err| AccessTokenError::Invalid(Box::new(err)))
+                .map(AccessTokenArg::KamuAccessToken);
+        }
```

Comment:
Why not to add the prefix to other type of tokens?
We can probably regenerate a token for Nusic manually without much trouble, other than this we don't have a need in maintaining format compatibility.

Reply:
Likely, Nusic will obtain this new type of token now anyway.

Reply:
I'm not sure what we will win with this changes? 
Now we will have `ka_token` and a simple JWT and it's easy to manage them